debug_toolbar/panels/sql/tracking.py

```diff
@@ -213,7 +213,6 @@ def _record(self, method, sql, params):
                         "trans_id": trans_id,
                         "trans_status": final_trans_status,
                         "iso_level": iso_level,
-                        "encoding": conn.encoding,
```

Comment:
@matthiask are we going to count this as a backwards incompatibility for any panel that might inherit from `SQLPanel`? It's been around since Pre v1.0, but I don't know of many things that inherit from `SQLPanel` or would use this.

Reply:
I don't think there is a backwards compatibility issue here since none of these attributes were ever being recorded, since prior to this PR the code was looking up the `vendor` attribute on the underlying DB connection rather than Django's `DatabaseWrapper`.  Thus none of the PostgreSQL-specific code was ever being run.

Reply:
I'm fine with this change. The variable is completely unused inside django-debug-toolbar. The variable has been added in 72271115974463f30a2f9f8755e42acc69e204c3 11 years ago but I'm not sure it has ever been used anywhere.

Reply:
That works for me. I wanted to make sure we made an active decision rather than letting it go silently. Thank you both!